common/src/main/java/org/jboss/hal/testsuite/creaper/command/UndeployCommand.java

```diff
@@ -48,7 +48,7 @@ public void apply(OnlineCommandContext ctx) throws Exception {
             if (fromAllGroups) {
                 cmd.append(" --all-relevant-server-groups");
             } else {
-                cmd.append("--server-groups").append(particularGroup);
+                cmd.append(" --server-groups= ").append(particularGroup);
```

Comment:
this change doesn't seem to be fully correct, I believe there shouldn't be space after the `=`. WDYT?